how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -0,0 +1,175 @@
+# How to use Kata Containers and "cri" (contaienrd plugin)
+## Requiriments 
+- Kuberentes, kubelet 1.10+
+- cri-containerd 1.1.0+
+- Kata Containers 0.2.0+
+
+## Run all this
+```
+#sed -n '/^```/,/^```/ p'  | sed '/^```/ d' > script.sh
+```
+
+## Install cri
+
+Follow the instructions from "cri" [installation guide](http://github.com/containerd/cri/blob/master/docs/installation.md)
+
+### Quick installation.
+```
+# Install cri-containerd
+VERSION="1.1.0"
```

Comment:
I need this to automate the testing for this documentation - I can remove it and query this before run this document (the code sections)

Reply:
Yes - it's a couple of extra lines but it would be good if you could query the version from https://github.com/kata-containers/runtime/blob/master/versions.yaml using https://github.com/mikefarah/yq.